packages/grid/_modules_/grid/hooks/features/infiniteLoader/useGridInfiniteLoader.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { optionsSelector } from '../../utils/optionsSelector';
+import { GridApiRef } from '../../../models/api/gridApiRef';
+import { useGridSelector } from '../core/useGridSelector';
+import { gridContainerSizesSelector } from '../../../components/GridViewport';
```

Comment:
Agree, it's used on a couple of more places but there isn't a good place to put it since the selectors are under the `hooks/` folder and that one doesn't really belong to a hook.